talaiot/src/test/kotlin/com/cdsap/talaiot/publisher/InfluxDbPublisherTest.kt

```diff
@@ -1,11 +1,10 @@
 package com.cdsap.talaiot.publisher
 
-import com.cdsap.talaiot.configuration.InfluxDbPublisherConfiguration
+import com.cdsap.talaiot.configuration.*
```

Comment:
I normally prefer specific imports instead of `*`. But this would be a project convention so please decide accordingly.

Reply:
done